### PR TITLE
x86: Implement vector comparison operations

### PR DESCRIPTION
**Pull request by BradleyWood**

### CODE REVIEW

**Comment by BradleyWood** [comment]
jenkins build all

**Comment by BradleyWood** [comment]
jenkins build all

**Comment by BradleyWood** [comment]
jenkins build all

**Review by 0xdaryl — APPROVED** [review]

File: compiler/x/codegen/OMRCodeGenerator.cpp [hunk 1]
```diff
@@ -1038,6 +1038,49 @@ bool OMR::X86::CodeGenerator::getSupportsOpCodeForAutoSIMD(TR::CPU *cpu, TR::ILO
             {
```
[the remark this answers is not in the document]
Comment (BradleyWood) [reply compiler/x/codegen/OMRCodeGenerator.cpp:1038]: Yes, it looks like these cases need to be moved around. I can open a fix on Monday.

**Pull request merged**